pkg/controller/cronjob/cronjob_controller_test.go

```diff
@@ -250,8 +250,6 @@ func TestSyncOne_RunOrNot(t *testing.T) {
 		"prev ran but done, long overdue, past short deadline, F":  {f, F, onTheHour, shortDead, T, F, weekAfterTheHour(), T, F, 1, 0},
 	}
 	for name, tc := range testCases {
-		name := name
-		tc := tc
```

Comment:
This has been introduced since the commit https://github.com/kubernetes/kubernetes/pull/89669
I think it is fine to remove here for cleanup.

/cc @yangkev
/lgtm

Reply:
I put this here like this defensively in case these tests were ever run in parallel 

https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables

Reply:
Thanks for your explanation, @yangkev  
I got the point, we need to keep those lines for goroutines.
I feel only thing we can do here is adding NOTE comment here to avoid deleting those lines in the future.

/lgtm cancel